sources/rest_api/__init__.py

```diff
@@ -232,6 +222,7 @@ def create_resources(
         (
             incremental_object,
             incremental_param,
+            incremental_cursor_transform,
```

Comment:
this value here is not accessible inside the `paginate_resource()` and `paginate_dependent_resource()`